unitTests/ui/RegisteredNamedMethodsTest.ts

```diff
@@ -209,14 +209,16 @@ export function RegisteredNamedMethodsTest() {
       });
 
       it('should allow to log a custom event', () => {
-        RegisteredNamedMethod.logCustomEvent(env.root, { name: 'foo', type: 'bar' }, {});
+        let fakeResult = FakeResults.createFakeResult();
```

Comment:
This could be made into a `const` instead

Reply:
ok!